src/envoy/tcp/mixer/tcp_filter.cc

```diff
@@ -253,6 +260,16 @@ class TcpInstance : public Network::Filter,
     data->duration = std::chrono::duration_cast<std::chrono::nanoseconds>(
         std::chrono::system_clock::now() - start_time_);
   }
+
+  std::string GetConnectionId() const override {
+    char connection_id_str[32];
+    memset(connection_id_str, 0, sizeof(connection_id_str));
+    StringUtil::itoa(connection_id_str, 32,
```

Comment:
if we can assume itoa append 0 at the end of string.  we don't need "memset".  could you double check?

Reply:
Yes, 0 is appended. https://github.com/envoyproxy/envoy/blob/master/source/common/common/utility.cc#L183
memset is removed.